.github/workflows/pathogen-repo-ci.yaml

```diff
@@ -255,22 +255,36 @@ jobs:
           runtime: ${{ matrix.runtime }}
 
       - name: Run ingest
+        id: ingest
         uses: ./.git/nextstrain/.github/actions/run-nextstrain-ci-build
         with:
           directory: ingest
           runtime: ${{ matrix.runtime }}
           artifact-name: ${{ inputs.artifact-name }}
 
       - name: Run phylogenetic
+        id: phylogenetic
         uses: ./.git/nextstrain/.github/actions/run-nextstrain-ci-build
         with:
           directory: phylogenetic
           runtime: ${{ matrix.runtime }}
           artifact-name: ${{ inputs.artifact-name }}
 
       - name: Run nextclade
+        id: nextclade
         uses: ./.git/nextstrain/.github/actions/run-nextstrain-ci-build
         with:
           directory: nextclade
           runtime: ${{ matrix.runtime }}
           artifact-name: ${{ inputs.artifact-name }}
+
+      - name: Verify a workflow ran
+        continue-on-error: false
```

Comment:
_nit_

Ditto re: `continue-on-error: false` being the default we rely on everywhere.